charts/temporal/templates/server-service.yaml

```diff
@@ -24,6 +24,11 @@ spec:
       {{- if hasKey .Values.server.frontend.service "nodePort" }}
       nodePort: {{ .Values.server.frontend.service.nodePort }}
       {{- end }}
+    - port: {{ .Values.server.frontend.service.httpPort }}
+      targetPort: http
+      protocol: TCP
+      name: http
+      # TODO: Also expose HTTP port
```

Comment:
Is this TODO still relevant? If so could you be a little more specific since it looks like this block is for exposing the HTTP port

Reply:
Fixed the comment, thanks.